extensions/interactions/GraphInput/directives/GraphInput.js

```diff
@@ -602,7 +602,7 @@ oppia.directive('graphViz', [
 
           var tryAddEdge = function(startIndex, endIndex) {
             if (
-                startIndex === null ||
+              startIndex === null ||
```

Comment:
I think I have understood the scope creation which appears to happen here. "||" creates no scope; instead, the indent rule measures 2-space indent from the beginning of the  "if" block.

Reply:
So I guess the next few lines should be de-indented by 2?

Reply:
When I put a paper on my computer screen to align startIndex vertically with the lines below, this is what I see:
```
            if (
              startIndex === null ||
                endIndex === null ||
                startIndex === endIndex ||
                startIndex < 0 ||
                endIndex < 0 ||
                startIndex >= $scope.graph.vertices.length ||
                endIndex >= $scope.graph.vertices.length) {
              return;
            }
```